src/compile/references.js

```diff
@@ -1,59 +1,54 @@
-var debug = require('debug')('velocity');
-
-module.exports = function(Velocity, utils) {
-
-  'use strict';
+module.exports = function (Velocity, utils) {
+  "use strict";
 
   /**
    * escapeHTML
    */
   function convert(str) {
+    if (typeof str !== "string") return str;
 
-    if (typeof str !== 'string') return str;
-
-    var result = ""
-    var escape = false
+    var result = "";
```

Comment:
It's a hassle, but we should not format files for 3rd party projects, whenever we want to merge upstream, we will run into trouble. Also, it's less clear what are our changes.